src/js/deckard.js

```diff
@@ -215,12 +215,13 @@
         $html.addClass(device.classes.join(' '));
     };
 
-    $window
-        .on(orientationEvent, function() {
-            $.extend($, orientation());
-            $html.toggleClass('portrait', $.orientation.portrait);
-            $html.toggleClass('landscape', $.orientation.landscape);
-        });
+    var updateOrientation = function() {
+        $.extend($, orientation());
+        $html.toggleClass('portrait', $.orientation.portrait);
+        $html.toggleClass('landscape', $.orientation.landscape);
```

Comment:
@scalvert it's kind of unfortunate how many times `toggleClass` touches the DOM:

* https://github.com/madrobby/zepto/blob/a9184b2c692aa68702e9024a4a72b2e47aeff036/src/zepto.js#L755-L764

Potentially a good place for optimization in future! It's too bad there isn't a performance minded toggleClass API.